web-ts/src/components/TuNewsList.tsx

```diff
@@ -46,12 +43,12 @@ class TunewsList extends React.PureComponent<PropsType> {
 
     return (
       <StyledList>
-        <InfiniteScroll loadMore={this.loadItems} hasMore={!isFetching && hasMore} loader={<LoadingSpinner key={0} />}>
```

Comment:
This looked just weird, imo better without a loading spinner. But feel free to check for yourself :)